rickshaw-run

```diff
@@ -857,6 +857,11 @@ sub source_container_image {
         # image needs to be deleted later, so save a reference to it here
         push(@local_images, $userenv_image);
     }
+    my $x = 0;
+    while ($x < $i) {
+        printf "Processing stage %d (%s)...\n\tReady\n", $x + 1, $workshop_args[$x]{'tag'};
```

Comment:
nit: $x + 1 ->  why do you iterate from 0 to $i and print $x + 1 ?

Reply:
It is fitting into the existing logic that surrounds this.  I think the rational is that its presenting to the user as layer 1, layer 2, layer 3, etc. while internally the array starts at 0.

https://github.com/perftool-incubator/rickshaw/blob/master/rickshaw-run#L822-L873